tests/unit/crsAdapter/TravelportCetsAdapter.test.js

```diff
@@ -366,7 +366,7 @@ describe('TravelportCetsAdapter', () => {
                 let service = '<Fah ServiceType="C" Key="VEHICLE.CODE/PICK.UP.LOCATION-DROP.OFF.LOCATION">' +
                     '<StartDate>01052017</StartDate>' +
                     '<Duration>4</Duration>' +
-                    '<Destination>PICK.UP.LOCATION</Destination>' +
+                    '<Destination>PIC</Destination>' +
```

Comment:
Wouldnt it make more sense as PUL other than PIC? (using the initial letters of each word) or was it a requirement?

Reply:
:D It was a requirement to use only the first 3 characters of the locations.